README.md

```diff
@@ -87,6 +87,10 @@ jsmediatags.read("http://www.example.com/music-file.mp3", {
   }
 });
 ```
+
+Note that URL has to include scheme(absolute URL).Relative URL is not supported for now.
```

Comment:
URL nomenclature is a bit tricky, you don't need a scheme for them to be absolute, but absolute URI's do! Here's a good article about it: https://dev.to/flippedcoding/what-is-the-difference-between-a-uri-and-a-url-4455

Here's my suggestion:
"Note that the URI has to include the scheme (e.g.: `https://`), as relative URIs are not supported."

Reply:
Got me : ) Thanks